src/McpBundle/Entity/OAuthAuthCode.php

```
@@ -0,0 +1,248 @@
+
+        return $this;
+    }
+
+    public function getUserIdentifier(): string|null
+    {
+        $id = $this->user->getId();
+
+        return $id?->toRfc4122();
+    }
+
+    public function setUserIdentifier(string $identifier): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$identifier'. (undefined)

(UnusedFormalParameter)